### PR TITLE
Fix typos & formatting

### PR DESCRIPTION
**Pull request by sebhoss**
Just a few small missing characters, upper vs. lower case and removal of unnecessary words.

### CODE REVIEW

**Comment by build-talend** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/5/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/5/)

File: README.md [hunk 1]
```diff
@@ -11,25 +11,25 @@ http://www.talend.com
...
 
 We welcome contributions of all kinds from anyone.
 
-Using the bug tracker [Talend bugtracker](http://jira.talendforge.org/) is the best channel for bug reports, feature requests and submitting pull requests.
+Using the [Talend bugtracker](http://jira.talendforge.org/) is the best channel for bug reports, feature requests and submitting pull requests.
 
 Feel free to share your Talend components on [Talend Exchange](http://www.talendforge.org/exchange).
 
-For code formatting, please use the configuration file and setup for eclipse or intelliJ that you find here : https://github.com/Talend/tools/tree/tools-java-formatter/0.1.0/tools-java-formatter
+For code formatting, please use the configuration file and setup for Eclipse or IntelliJ that you find here : https://github.com/Talend/tools/tree/tools-java-formatter/0.1.0/tools-java-formatter
```
Comment (RyanSkraba) [inline README.md]: Another French colon! :)

Also, we should probably point them to the master branch: https://github.com/Talend/tools/tree/master/tools-java-formatter

File: README.md [hunk 2]
```diff
@@ -11,25 +11,25 @@ http://www.talend.com
 ## Contents
 
 This repository contains the source files of shared libraries for all products.
```
Comment (RyanSkraba) [inline README.md]: This doesn't mean anything, and is probably wrong as well.

This repository contains commonly used classes that are expected to be shared across all Talend products. (?)

File: README.md [hunk 3]
```diff
@@ -11,25 +11,25 @@ http://www.talend.com
...
-You can ask for help on our [Forum](http://www.talend.com/services/global-technical-support).
+You can ask for help on our [forum](http://www.talend.com/services/global-technical-support).
 
 
 ## Contributing
 
 We welcome contributions of all kinds from anyone.
 
-Using the bug tracker [Talend bugtracker](http://jira.talendforge.org/) is the best channel for bug reports, feature requests and submitting pull requests.
+Using the [Talend bugtracker](http://jira.talendforge.org/) is the best channel for bug reports, feature requests and submitting pull requests.
```
Comment (RyanSkraba) [inline README.md]: Update to https://jira.talendforge.org/projects/TDKN for Daikon-specific bug reports.

JIRA is not a very good place to submit pull requests :P  Probably https://github.com/Talend/daikon.

File: README.md [hunk 4]
```diff
@@ -11,25 +11,25 @@ http://www.talend.com
...
 
 Feel free to share your Talend components on [Talend Exchange](http://www.talendforge.org/exchange).
```
Comment (RyanSkraba) [inline README.md]: Maybe... but irrelevant to Daikon.  Remove this line?

**Comment by RyanSkraba** [comment]
Oh my... found another problem in the readme :)  Can you fix the typo in the last line?

> "The Daikon is a root vegetable and since this project is supposed to be the root of many projects we thought is was a fun and appropriate name."

My apologies -- thanks for this cleanup work!

**Comment by build-talend** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/8/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/8/)

File: README.md (hunk 1 above)
Comment (sebhoss) [inline README.md]: Yep fixed

File: README.md (hunk 2 above)
Comment (sebhoss) [inline README.md]: Agreed - sounds better!

File: README.md (hunk 3 above)
Comment (sebhoss) [inline README.md]: Update JIRA link + now pointing to GitHub for pull requests

File: README.md (hunk 4 above)
Comment (sebhoss) [inline README.md]: removed in latest change

**Comment by sebhoss** [comment]
No problem :-) - I've added another list entry for "daikon-service" & I'm not sure if the "service" module needs to be mentioned as well. Do you know anything about that?

**Comment by RyanSkraba** [comment]
LGTM!  I think it's safe to say you can merge this, since it's an obvious improvement :+1: !

I don't think there's anything to say about the service module here -- @fhuaulme: what is it meant for?

**Pull request merged**